pendingTranslations.ftl

```diff
@@ -11,3 +11,27 @@ main-nav-link-faq-tooltip = Frequently asked questions
 
 footer-external-link-faq-label = FAQs
 footer-external-link-faq-tooltip = Frequently asked questions
+
+# Status Pill
+
+status-pill-action-needed = Action Needed
+status-pill-progress = In Progress
+status-pill-fixed = Fixed
+
+# Exposure Card
+
+exposure-card-family-members = Family members
+exposure-card-email = Type
```

Comment:
Q: should this be "Type" or "Email"?

Storybook:
![Screenshot 2023-06-26 at 4 27 04 PM](https://github.com/mozilla/blurts-server/assets/557895/9b139489-ff57-4669-bcfb-cc8f16123970)

Figma:
https://www.figma.com/file/CaEKIhvSJqf6KNIMzSkt40/Concepts-for-Monitor-MVP-Redesign?type=design&node-id=546-44852&mode=design&t=bk52u2JrpN1imc6f-0